src/test/ui/self/invalid_dispatch_from_dyn_impls.rs

```diff
@@ -1,3 +1,6 @@
+// revisions: rustc chalk
```

Comment:
Hmm... I only realize now that these tests don't do anything, I think...? Not sure how we can test these though.